libvips/iofuncs/init.c

```diff
@@ -270,31 +270,26 @@ vips_load_plugins( const char *fmt, ... )
 	if( !(dir = g_dir_open( dir_name, 0, NULL )) )
 		return;
 
-        while( (name = g_dir_read_name( dir )) )
-                if( vips_ispostfix( name, "." G_MODULE_SUFFIX )
-#if ENABLE_DEPRECATED
-				|| vips_ispostfix( name, ".plg" ) 
-#endif
-			) { 
```

Comment:
I think we could safely remove this (i.e. build modules as `*.dylib` on macOS):
https://github.com/libvips/libvips/blob/00e4047ff9a223132adfdf874f723139e399c6f2/libvips/meson.build#L75-L80
Now that we no longer use the deprecated `G_MODULE_SUFFIX` macro. Let me open a follow-up PR for this.

Reply:
#3554